app/controllers/responsible_body/extra_mobile_data_requests_controller.rb

```diff
@@ -26,6 +26,9 @@ def create
         # clear the stashed params once the user has confirmed them
         session.delete(:extra_mobile_data_request_params)
         @extra_mobile_data_request.save!
+
+        NotifyExtraMobileDataAccountHolderService.call(@extra_mobile_data_request)
+
```

Comment:
Probably ok to keep these here for now, but I wonder whether the request saving and notification sending should live in a separate service, and not in the controller...